index.html

```diff
@@ -155,8 +256,8 @@ <h3 class="pcol">LET’S CONNECT</h3>
                             <img class="imageabout" src="csss.png" alt="Js"> <span class="lastdiv">HTML</span>
                         </div>
                     </li>
```

Comment:
- [ ] Please 😃try to add the background image of the form at the correct position, so please fix the position of it to be like the Figma Design😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/215076477-6f6a476a-95b7-43a9-8659-d780eb04a4a7.png) |    ![image](https://user-images.githubusercontent.com/83360636/215076621-a0ecaa51-939a-48fa-af18-cbd80c868db6.png)|

Reply:
suggested changes made. Screen shot attached for your kind reference below.
![image](https://user-images.githubusercontent.com/117091923/215103822-7e49463b-e525-4643-a38a-9eac74f32b7a.png)